app/models/legislator.rb

```diff
@@ -122,7 +122,7 @@ def representative?
   def name
     first = verified_first_name || nickname || first_name
     last  = verified_last_name  || last_name
-    first + ' ' + last
+    first + ' ' + last + (name_suffix ? ', ' + name_suffix : '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.